Backend/src/main/java/project/airbnb/bnb/Bnb.java

```diff
@@ -1,15 +1,63 @@
 package project.airbnb.bnb;
 
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
 import javax.persistence.Column;
+import javax.persistence.Embedded;
 import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import project.airbnb.bnbImage.BnbImage;
 
 @Entity
+@Getter // 임시 생성 (BC)
+@AllArgsConstructor // 임시 생성 (BC)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class Bnb {
 
-	@Id @GeneratedValue
+	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
 	@Column(name = "bnb_id")
 	private Long id;
 
+	@OneToMany(mappedBy = "bnb", cascade = CascadeType.ALL)
+	private List<BnbImage> images = new ArrayList<>();
+
+	private String name;
+	private double rating;
+	private int reviewCount;
+	@Embedded
+	private Address address;
+
+	@Enumerated(EnumType.STRING)
+	private BnbType bnbType;
+
+	private String host;
+
+	private Integer maxGuestNumber;
+
+	@Embedded
+	private Time time;
+
+	@Embedded
+	private BnbOption bnbOption;
+
+	private String description;
+
+	private Long fee;
```

Comment:
개행도 컨벤션 중 하나입니다. 일관적인 사용을 해주세요.

`nit`
저는 `Embedded`를 쓰기 애매한 경우 비슷한 업무 단위로 묶고 개행으로 구분하고 주석을 달곤 합니다.

```suggestion
        // 중요 정보
        @Enumerated(EnumType.STRING)
	private BnbType bnbType;
	private String name;
	@Embedded
	private Address address;
	private Long fee;
	private Integer maxGuestNumber;	

        // 비 중요 정보
	@Embedded
	private Time time;
	private String host;
	@Embedded
	private BnbOption bnbOption;

        // 기타
	private String description;
	private double rating;
	private int reviewCount;
```